apps/iframe/src/services/types.ts

```diff
@@ -0,0 +1,93 @@
+export type RenderTypes = undefined | 'account' | 'migration' | 'authenticate' | 'migration_fail'
```

Comment:
Aren't these types already provided by another package? If not, might be better to create a new package for shared types to avoid possible errors/mismatches with duplication.

Reply:
I don't think that these are exported anywhere else, I checked through js-glaze, self-id & 3id-connect and none of them make use of the types here. 

There's definitely a use case for exporting them to a new package however I'm fairly sure that they're only used by 3ID-Connect and specifically the iFrame application so I'm less convinced that another package is the right way to go since it all seems fairly use-case specific.

Reply:
Sorry, I didn't check the full file. I was only looking at the top line. I'll go through & find any types and minimize duplication.

Reply:
Fun discovery, none of the types were in use so I've removed all but the RenderTypes that I should be using in the actual app. I'll implement them before EoD